wgs/cnai/glossary/index.md

```diff
@@ -9,6 +9,18 @@ Defines key terms used in the Cloud Native Working Group’s writings.
 
 See also: [CNCF Glossary](https://glossary.cncf.io/)
 
+## AI for CN
+
+A class of tools integrates AI with the intent to optimize CN system management outcomes. Optimization can take different forms, from actionable advice to agentic actions.
+
+## CN for AI
+
+A of tools integrates AI tooling effectively enabling AI workloads. Effectively can mean a reduction in time, cost, and complexity when designing and hosting these workloads.
+
+## CNAI
+
+CNAI (Cloud Native Artificial Intelligence) describes the tooling and techniques around use of Cloud Native tooling in support of AI.
```

Comment:
suggest a change to "CNAI (Cloud Native Artificial Intelligence) describes the tooling and techniques that support the  deployment and operation of AI on Cloud Infrastructure"

Reply:
Suggestion: I think we should also include a line on how we want to explore how AI can benefit from Cloud Native and vice versa

Reply:
Suggestion: as what @rajaskakodkar mentioned, CN Toolings basically support implementation of AI inside Cloud Native, so I believe those purpose should be mentioned to show the correlation of AI and Cloud Native